### PR TITLE
Add padding to prevent music controls from clipping

### PR DESCRIPTION
**Pull request by albeeeert**
**Changes**
Add bottom padding for artist view and e.g. photos view to prevent music controls from clipping with information. Removes a needless inline style from list.html.

**Issues**
Fixes issue #2314.

### CODE REVIEW

File: src/assets/css/site.scss [hunk 1]
```diff
@@ -81,7 +81,8 @@ div[data-role="page"] {
 .content-primary,
 .padded-bottom-page,
 .page,
-.pageWithAbsoluteTabs .pageTabContent {
+.pageWithAbsoluteTabs .pageTabContent,
+.itemsContainer {
```
Comment (dmitrylyzo) [inline src/assets/css/site.scss:85]: `itemContainer` is too generic - all containers obtain bottom padding. Instead, you can add `padded-bottom-page` to `detailPageWrapperContainer` on music item details page.

File: src/controllers/list.html [hunk 2]
```diff
@@ -48,7 +48,7 @@
                 <span class="material-icons more_vert"></span>
             </button>
         </div>
-        <div is="emby-itemscontainer" class="vertical-wrap itemsContainer centered" style="padding: 0 0 10vh 0;">
+        <div is="emby-itemscontainer" class="vertical-wrap itemsContainer centered">
```
Comment (dmitrylyzo) [inline src/controllers/list.html:51]: Agree with this. Also I would add mentioned `padded-bottom-page` to `pageContainerTopPadding` element as above for the same purpose.

File: src/assets/css/librarybrowser.css [hunk 3]
```diff
@@ -1052,7 +1052,8 @@ div.itemDetailGalleryLink.defaultCardBackground {
 }
 
 .itemsContainer {
-    margin: 0 auto;
+    margin-left: auto;
+    margin-right: auto;
```
Comment (dmitrylyzo) [inline src/assets/css/librarybrowser.css:1056]: With `padded-bottom-page` class, this may be left as it was.

**Comment by dkanada** [comment]
@albeeeert did you have any comments about the suggestions?

File: src/assets/css/site.scss (hunk 1 above)
Comment (albeeeert) [reply src/assets/css/site.scss]: Yeah, I agree. I amended as you said, and it seems to work.

**Comment by albeeeert** [comment]
Made changes as @dmitrylyzo stated in the latest commit. Seems to work and I agree that the itemsContainer was way too generic.

**Review by dmitrylyzo — APPROVED** [review]

**Review by dkanada — APPROVED** [review]

**Pull request merged**